tests/Unit/Bindings/Browser/ObjectServiceTest.php

```diff
@@ -291,7 +294,16 @@ public function createDocumentDataProvider()
             'Create document without stream' => array(
                 Url::createFromUrl(
                     self::BROWSER_URL_TEST
-                    . '?propertyId[0]=cmis:name&propertyValue[0]=name.jpg&cmisaction=createDocument&succinct=false'
+                ),
+                array(
+                    'cmisaction' => 'createDocument',
+                    'succinct' => 'false',
+                    'propertyId' => array(
```

Comment:
The parameters when passed as array generate the same `param[0]=value` format so it should be correct this way, too.